nodebuilder/das/config.go

```diff
@@ -0,0 +1,70 @@
+package das
+
+import (
+	"errors"
+	"time"
+)
+
+// Config contains configuration parameters for the DASer (or DASing process)
+type Config struct {
+	//  SamplingRange is the maximum amount of headers processed in one job.
+	SamplingRange uint
+
+	// ConcurrencyLimit defines the maximum amount of sampling workers running in parallel.
+	ConcurrencyLimit int
+
+	// BackgroundStoreInterval is the period of time for background checkpointStore to perform a checkpoint backup.
+	BackgroundStoreInterval time.Duration
+
+	// PriorityQueueSize defines the size limit of the priority queue
+	PriorityQueueSize int
```

Comment:
What about making these a units?

Reply:
For these, since they are used mostly in comparisons against slices lens (_`len(slice) > concurrencyLimit`, similarly for priorityQueueSize_) it makes more sense to keep them as ints to avoid conversions.